### PR TITLE
perf: send seed data requests asynchronously

### PR DESCRIPTION
**Pull request by mcmcgrath13**
# PULL REQUEST

## Summary
Use asynchronous requests when converting seed data. Locally, this speeds up conversion from 8:45 to 5:12

* Split the `create-seed-data-py` script into two phases: 1) gathering all the requests to send, 2) sending all the requests async and handling responses
* This also adds `n of total` printing, which I find a nicer ux

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/seed-scripts/create-seed-data.py
Comment (BobanL) [inline containers/ecr-viewer/seed-scripts/create-seed-data.py:1]: I'm not sure on other machines, but locally I'm actually getting slower results.

Async: 10:09:11 - 10:12:45 = 3 minutes 34 seconds
Sync: 10:16:11 - 10:19:36 = 3 minutes 25 seconds

Comment (austin-hall-skylight) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py:1]: Async ran for me in 5:17 while sync was 6:10.

Comment (mcmcgrath13) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py:1]: so almost a minute improvement, not great, not awful

Should we go ahead and merge? or any concerns about adding the async-ness in general?

Comment (BobanL) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py:1]: It looks good to me! I was testing around with lower memory / cpu and got similar results to Mary's!

**Review by BobanL — APPROVED** [review]

**Pull request merged**